src/dmd/func.d

```diff
@@ -1419,6 +1420,14 @@ extern (C++) class FuncDeclaration : Declaration
         }
         else if (isSafe())
             return true;
+        else if (global.params.showSystem &&
+                 type.toTypeFunction().trust == TRUST.default_ &&
+                 !(flags & FUNCFLAG.warnedSystem))
+        {
+            flags |= FUNCFLAG.warnedSystem;
+            message(loc, "%s is implicitly @system but not marked as @system", ident.toChars);
```

Comment:
Still missing some of the changes mentioned before: Use backtips around keywords and identifiers / fully qualified names to make them stand out (they get colored). 
Also prefer using `"%s %s"` / `kind(), toPrettyChars()` over just `ident`, as it's both more consistent with what the rest of the compiler does and makes the error unambiguous. We have plenty of heavily templated / mixed in code that would result in very unhelpful messages otherwise.

Reply:
Switched to `toPrettyChars` and added the backticks but used `function` instead of `kind` since this is limited to functions anyway.